suite-native/alerts/src/alertsAtoms.ts

```diff
@@ -9,13 +9,13 @@ import { ButtonColorScheme, PictogramVariant } from '@suite-native/atoms';
 export type Alert = RequireAllOrNone<
     {
         title: ReactNode;
-        description: string;
+        description: ReactNode;
```

Comment:
I am not really happy with ReactNode, it can be undefined too which is not great. Maybe there is better way how to type it? @PeKne

Reply:
I don't know about any built-in React type that do that out of do box. What about excluding nullish values out of the `ReactNode`?
eg.
```suggestion
        description: NonNullable<ReactNode>;
```

Reply:
FYI an array of nullable values could be still possibly passed there.

To fix that we would need to create a bit more complicated type:
```tsx
type DefinedNode = Exclude<NonNullable<ReactNode>, Iterable<ReactNode>> | Iterable<DefinedNode>;
```

Reply:
Just a thought (from [this article](https://www.totaltypescript.com/type-safe-children-in-react-and-typescript)):
![image](https://github.com/trezor/trezor-suite/assets/36101761/ca40cf6f-0809-4071-af97-cc6819e015c1)

I wouldn't mind leaving `ReactNode` here but it's just my 2 sats for discussion.

Reply:
The description (and title) are passed into `<Pictogram/>` and that is ready to handle even `undefined`

I think it can be used as `ReactNode` here

Reply:
I believe using `ReactNode` is mostly okay and in our case way better than `string`. I would use it "by default" and in places where we think it makes sense, we can use a more restrictive type so that undefined doesn't go there. I would use `ReactElement` in such places (I would not allow either string or iterable).

Reply:
Ok, let's not overthink it and use ReactNode